packages/run-protocol/src/reserve/assetReserve.js

```diff
@@ -51,9 +51,22 @@ const start = async (zcf, privateArgs) => {
   /**
    * Used to look up the brands for keywords, excluding RUN because it's a special case.
    *
-   * @type {MapStore<Keyword, [brand: Brand, liquidityBrand: Brand]>}
```

Comment:
@Chris-Hibbert given the requirement that `AddCollateral` invitation accept any brand, I thought it cleanest to make these maps ignore the liquidity distinction. instead that's handled through a separate lookup between brands.